src/applications/simple-forms/20-10207/tests/e2e/10207-pp.cypress.spec.js

```diff
@@ -52,6 +54,41 @@ const testConfig = createTestConfig(
             .click();
         });
       },
+      'mailing-address': ({ afterHook }) => {
+        afterHook(() => {
+          cy.get('@testData').then(data => {
+            cy.get(
+              'va-segmented-progress-bar[uswds][heading-text][header-level="2"]',
+            )
+              .should('be.visible')
+              .then(() => {
+                cy.get('[name="root_mailingAddress_state"]')
+                  .should('not.have.attr', 'disabled')
+                  .then(() => {
+                    // callback to avoid field-disabled errors, but
+                    // even now we must wait a bit!
+                    // eslint-disable-next-line cypress/no-unnecessary-waiting
+                    cy.wait(1000);
```

Comment:
the workaround we've usually done for addresses, it is to fill out the pattern, and then manually fill out the field that had changed. Would that work here instead? rather than using a wait

Reply:
I'll refactor this in the next PR; I think my code above was before I discovered your web-component helpers.